src/utils/plotSpecs/generateHeatmapSpec.js

```diff
@@ -1,45 +1,45 @@
-const generateSpec = (config, groupName) => {
-  let legend = [];
-  if (config.legend.position === 'horizontal') {
-    legend = [
-      {
-        fill: 'color',
-        type: 'gradient',
-        orient: 'bottom',
-        direction: 'horizontal',
-        title: ['Intensity'],
-        labelFont: { value: config.fontStyle.font },
-        titleFont: { value: config.fontStyle.font },
-        gradientLength: {
-          signal: 'width',
-        },
+const generateSpec = (config) => {
```

Comment:
is this spec used for both heatmaps (data exploration and on plots and tables)?

Reply:
No the data-exploration heatmap spec is different (has tooltips, no second legend, mouseOver events)

Reply:
no, I meant whether this file "src/plotSpecs/generateHeatmapSpec.js" is used for generating both the data exploration and the plots and tables heatmap

Reply:
No it is only for plots and tables heatmap